### PR TITLE
dwm: new package at 6.2

### PR DESCRIPTION
**Pull request by fanboimsft**
dynamic window manager from suckless.org

- [6.2] Latest upstream version.

### CODE REVIEW

File: community/dwm/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+# Amend install path to '/usr'
```
Comment (dylanaraps) [inline community/dwm/build]: Full stop `.`.

File: community/dwm/build [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+sed -i'' "s#/usr/local#/usr#" config.mk
+
+# Remove pixel gap
```
Comment (dylanaraps) [inline community/dwm/build]: Full stop `.`.

File: community/dwm/build [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+sed -i'' "s/static const int resizehints = 1/static const int resizehints = 0/" config.def.h
```
Comment (dylanaraps) [inline community/dwm/build]: Any specific reason for this change?

Comment (fanboimsft) [reply community/dwm/build]: Without the patch there is a weird 1pixel gap at the border of some windows. With the patch all windows just fill the screen as they should.

Comment (dylanaraps) [reply community/dwm/build]: :+1:

**Comment by dylanaraps** [comment]
LGTM :+1:

**Pull request merged**